.github/workflows/torchbench-nightly-mps.yml

```diff
@@ -0,0 +1,48 @@
+name: torchbench-nightly-mps
+
+# TODO - this is actually meant to be a nightly run, it's only ciflow workflow for initially debugging
+# Please do not merge
+on:
+  push:
+    tags:
+      - ciflow/mps/*
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  macos-py3-arm64-mps-torchbench-test:
+    name: macos-py3-arm64-mps-torchbench-test
+    uses: ./.github/workflows/_mac-build.yml
+    with:
+      sync-tag: macos-py3-arm64-mps-torchbench-test
+      build-environment: macos-py3-arm64
+      runner-type: macos-m1-stable
+      build-generates-artifacts: true
+      # To match the one pre-installed in the m1 runners
+      python-version: 3.9.12
+      # The runner macos-m2-15 is not a typo, it's a custom runner that is different
+      # than our AWS macos-m1-14 runners
+      test-matrix: |
+        { include: [
+          { config: "mps", shard: 1, num_shards: 1, runner: "macos-m2-15" },
+        ]}
+    secrets:
+      HUGGING_FACE_HUB_TOKEN: ${{ secrets.HUGGING_FACE_HUB_TOKEN }}
+
+  macos-py3-arm64-mps-torchbench-test:
+    name: macos-py3-arm64-mps-torchbench-test
+    uses: ./.github/workflows/_mac-test-mps.yml
```

Comment:
Is this running torchbench or just test_mps.py nightly? I think if you want torchbench, it should rather be in https://github.com/pytorch/benchmark?

Reply:
Having it in the pytorch CI will let people optionally trigger perf runs on their PRs, which could be useful